Services/include/communication/communication_service.h

```diff
@@ -27,23 +27,24 @@
 #define MAX_FREQ = 2290
 
 typedef enum {
-    S_GET_TEMP = 0,
-    S_GET_FREQ = 1,
-    S_GET_CONTROL = 2,
-    S_GET_ENCODER = 3,
-    S_GET_PA_POWER = 4,
-    S_GET_STATUS = 5,
-    S_GET_TR = 6,
-    S_GET_BUFFER = 7,
-    S_GET_HK = 8,
-    S_SOFT_RESET = 9,
-    S_GET_FULL_STATUS = 10,
-    S_SET_FREQ = 11,
-    S_SET_CONTROL = 12,
-    S_SET_ENCODER = 13,
-    S_SET_PA_POWER = 14,
-    S_GET_CONFIG = 15,
-    S_SET_CONFIG = 16
+  S_GET_TEMP = 0,
+  S_GET_FREQ = 1,
+  S_GET_CONTROL = 2,
+  S_GET_ENCODER = 3,
+  S_GET_PA_POWER = 4,
+  S_GET_STATUS = 5,
+  S_GET_FW = 6,
+  S_GET_TR = 7,
+  S_GET_BUFFER = 8,
+  S_GET_HK = 9,
+  S_SOFT_RESET = 10,
+  S_GET_FULL_STATUS = 11,
+  S_SET_FREQ = 12,
+  S_SET_CONTROL = 13,
+  S_SET_ENCODER = 14,
+  S_SET_PA_POWER = 15,
+  S_GET_CONFIG = 16,
+  S_SET_CONFIG = 17
```

Comment:
Has this been updated in the groundstation code as well?

Reply:
I'm not sure it has been committed yet, but yes it exists. @tganley wanna check this?

Reply:
Fixed now https://github.com/AlbertaSat/ex2_ground_station_software/commit/6fc9c4f95307cad565db6ad6daf224d0a97b1df1